Recrutify/Recrutify.Services/Services/ProjectService.cs

```diff
@@ -90,6 +91,7 @@ public async Task<ProjectDTO> UpdateAsync(UpdateProjectDTO projectDto)
             newProject.Managers = projectDto.Managers.GetStaff(users);
             newProject.Mentors = projectDto.Mentors.GetStaff(users);
             newProject.Recruiters = projectDto.Recruiters.GetStaff(users);
+            newProject.CurrentApplicationsCount = currentProject.CurrentApplicationsCount;
```

Comment:
delete it, and write on  87      var newProject = _mapper.Map<Project>(projectDto, currentProject.DeepCopy());

Reply:
done